pkg/operator/ceph/cluster/controller.go

```diff
@@ -218,15 +222,16 @@ func add(mgr manager.Manager, r reconcile.Reconciler, context *clusterd.Context)
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
 func (r *ReconcileCephCluster) Reconcile(context context.Context, request reconcile.Request) (reconcile.Result, error) {
 	// workaround because the rook logging mechanism is not compatible with the controller-runtime logging interface
-	reconcileResponse, err := r.reconcile(request)
-	if err != nil {
+	reconcileResponse, cephCluster, err := r.reconcile(request)
+	if err != nil && !kerrors.IsNotFound(err) {
```

Comment:
I think we still want to log the error even if the cephCluster is not found.

Reply:
Done